.travis.yml

```diff
@@ -0,0 +1,16 @@
+sudo: false
+language: go
+go:
+  - 1.x
```

Comment:
If you want to support older versions of Go and test them in CI, we can add them here.

Reply:
I don't think that's necessary yet.